.gitignore

```diff
@@ -4,7 +4,7 @@
 **/*.npy
 **/*.png
 **/*.gif
-**/*.cpp
+pymoo/cython/*.cpp
```

Comment:
Previously there was a cpp file under `pymoo/cython/vendor` that was also git-ignored. We only want to ignore the cython-generated files.